cmd/gardener-controller-manager/app/gardener_controller_manager.go

```diff
@@ -173,6 +173,8 @@ These so-called control plane components are hosted in Kubernetes clusters thems
 		RunE: func(cmd *cobra.Command, args []string) error {
 			verflag.PrintAndExitIfRequested()
 
+			cmd.SilenceUsage = true
```

Comment:
Can we instead set this field on the Command itself such as

```go
	cmd := &cobra.Command{
			// omitted
			SilenceUsage: true,
			// omitted
	}
```

Reply:
Yes, but GAC and GSAC have used it in `RunE` so for consistency I followed the same suite.

Reply:
Sure. I would rather vote to adapt to the above suggestion.

Reply:
I have updated it. PTAL